server/src/controllers/christmas-tree.es6

```diff
@@ -215,6 +237,60 @@ const permitResult = (permit, svgData) => {
   return result;
 };
 
+const returnSavedPermit = (res, savedPermit, svgData) => {
+  return res.status(200).send(permitResult(savedPermit, svgData));
+};
+
+const parseXMLFromPayGov = (res, xmlResponse, permit) => {
```

Comment:
similarly this seems like it could be broken down into several functions.

Reply:
Yea, created separate functions.